### PR TITLE
General cleanup

### LINKED ISSUE #6: consolidate and auto-close build/deploy/launch dialog(s)
When you launch a Dartino or SoD app, you see several dialogs and they "build up" over time as you repeatedly write code - launch - repeat. Dialogs from prior launches should auto-close during a new launch and the separate build, deploy, and launch dialogs should be consolidated.

### LINKED ISSUE #15: menu item to open SDK docs
"Open SDK docs" would open browser on `<dartino-sdk>/docs/index.html`

### PR DESCRIPTION
**Pull request by danrubel**
- menu actions to open Getting Started and SDK docs - fixes #15
- auto close/cleanup compile/launch dialogs - fixes #6
- change imports within same lib directory tree to be relative

@devoncarew

### CODE REVIEW

File: lib/dartino.dart [hunk 1]
```diff
@@ -176,3 +180,14 @@ _runAppOnDevice(event) async {
     atom.notifications.addInfo('Launched app on device', dismissable: true);
   }
 }
+
+_showGettingStarted(event) async {
```
Comment (devoncarew) [inline lib/dartino.dart]: unneeded `async`

File: lib/dartino.dart [hunk 2]
```diff
@@ -176,3 +180,14 @@ _runAppOnDevice(event) async {
...
+  shell.openExternal('https://dartino.org/index.html');
+}
+
+_showSdkDocs(event) async {
```
Comment (devoncarew) [inline lib/dartino.dart:189]: ditto re: `async`

File: lib/dartino.dart [hunk 3]
```diff
@@ -176,3 +180,14 @@ _runAppOnDevice(event) async {
...
+  Sdk sdk = await findSdk(null);
+  if (sdk == null) return;
+  // TODO(danrubel) convert Windows file path to URI
+  shell.openExternal('file://${sdk.sdkRootPath}/docs/index.html');
```
Comment (devoncarew) [inline lib/dartino.dart]: You may want to look at `Uri.file(String path, {bool windows})` here (but I'd verify first that it does what you want).

**Comment by devoncarew** [comment]
lgtm :)

File: lib/dartino.dart (hunk 1 above)
Comment (danrubel) [inline lib/dartino.dart]: Removed.

File: lib/dartino.dart (hunk 2 above)
Comment (danrubel) [inline lib/dartino.dart:189]: It is needed here because of the `await` on the next line.

File: lib/dartino.dart (hunk 3 above)
Comment (danrubel) [inline lib/dartino.dart]: That's what I needed. Thanks!

File: lib/dartino.dart (hunk 2 above)
Comment (devoncarew) [inline lib/dartino.dart:189]: ah, I missed that

**Comment by danrubel** [comment]
Merged https://github.com/dartino/atom-dartino/commit/a68b6861925fceab8583f42123f69995e0852b9d

**Pull request closed without merging**